scripts/coverage_report.sh

```diff
@@ -65,9 +65,22 @@ if [ $SKIP_PROMPTS -eq 0 ]; then
 fi
 
 # Set the flags necessary for coverage output
-export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
-export RUSTC_BOOTSTRAP=1
-export CARGO_INCREMENTAL=0
+# export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
+# export RUSTC_BOOTSTRAP=1
+# export CARGO_INCREMENTAL=0
+
+# env generated by `cargo llvm-cov show-env` on current CI server
+# ----------------------------------------
+# RUSTFLAGS=" -C instrument-coverage --cfg coverage --cfg trybuild_no_target"
+# LLVM_PROFILE_FILE="/runner/_work/starcoin/starcoin/target/starcoin-%m.profraw"
+# CARGO_INCREMENTAL="0"
+# RUST_TEST_THREADS="1"
+# CARGO_LLVM_COV_TARGET_DIR="/runner/_work/starcoin/starcoin/target"
```

Comment:
not tested which of these env makes the coverage test faster, but it does work

Reply:
@taiki-e could you take a look ?

Reply:
`-Zprofile`, `-Ccodegen-units=1`, and `-Clink-dead-code` are unneeded for cargo-llvm-cov and removal of them may improve performance.

Reply:
thanks for this info